piplicenses.py

```diff
@@ -93,13 +93,12 @@
     "License",
 )
 
-
-METADATA_KEYS = (
-    "home-page",
-    "author",
-    "license",
-    "summary",
-)
+METADATA_KEYS = {
+    "home-page": ["home-page"],
+    "author": ["author", "author-email"],
+    "license": ["license"],
+    "summary": ["summary"],
+}
```

Comment:
I understand that this is unavoidable in order to handle `"author-email"` well, but this structure is redundant.

I have no idea how to write smarter either...

Reply:
I could implement a `Union[str, list[str]]` approach with the following logic

```python
for field_name, field_selectors in METADATA_KEYS.items():
    value = None
    if isinstance(field_selectors, list):
        for selector in field_selectors:
            value = metadata.get(selector, None)
                if value:
                    break
    else:
        value = metadata.get(field_selectors, None)
    pkg_info[field_name] = value or LICENSE_UNKNOWN
```

but i think this makes it even more fuzzy :laughing:

Reply:
Hm... yes, indeed 😅